include/vinecopulib/bicop/implementation/class.ipp

```diff
@@ -880,19 +966,20 @@ Bicop::fit(const Eigen::MatrixXd& data, const FitControlsBicop& controls)
 //!
 //! @param data An \f$ n \times (2 + k) \f$ matrix of observations contained in
 //!   \f$(0, 1) \f$, where \f$ k \f$ is the number of discrete variables.
-//! @param controls The controls (see `FitControlsBicop`).
+//! @param controls The controls (see `FitControls`).
 inline void
-Bicop::select(const Eigen::MatrixXd& data, FitControlsBicop controls)
+Bicop::select(const Eigen::MatrixXd& data, FitControls& controls)
```

Comment:
Wouldn't it be better to just pass the controls by value/copy? The current version is modifying the user's controls while the function is running. Seems too risky for a function that is likely called in parallel.